src/strings/csv/en.csv

```diff
@@ -1670,6 +1670,7 @@ TIME_ZONE,Time Zone,
 TIME_ZONE_REQUIRED,Time Zone *,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 774]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Can probably remove all the MODULE_N stuff<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2570?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->